libpod/container_api.go

```diff
@@ -815,6 +820,6 @@ func (c *Container) Restore(ctx context.Context, options ContainerCheckpointOpti
 			return err
 		}
 	}
-
+	defer c.newContainerEvent(events.CheckpointRestore)
```

Comment:
Maybe rename from CheckpointRestore to RestoreFromCheckpoint (or just Restore)?

Reply:
Just Restore